hack/ocp-rebase.sh

```diff
@@ -8,24 +8,52 @@
 
 set -e
 
+# make sure we have downstream and upstream trees before the script starts
+if ! git remote show | grep -q ^upstream$; then
```

Comment:
After running the script myself something else occurs to me: We should use the same logic as the upstream_remote and downstream_remote commands below on line 22 and 23. It's not the name of the remotes that we care about, it's the address.